README.md

```diff
@@ -18,6 +18,8 @@ gem 'solidus_related_products'
 Run:
 ```sh
 $ bundle install
+$ bundle exec rake railties:install:migrations
+$ bundle exec rake db:migrate
```

Comment:
This should be done by the install script already: https://github.com/solidusio-contrib/solidus_related_products/blob/master/lib/generators/solidus_related_products/install/install_generator.rb#L11

Did you have issues with that?